.flowconfig

```diff
@@ -1,6 +1,7 @@
 [ignore]
 .*/node_modules/react-native/.*
 .*/node_modules/fbjs/.*
+node_modules/config-chain/test/broken.json$
```

Comment:
this location in this file is a bit odd. Can you sort it alphabetically (after the dot, as last item)?